components/export/ui/src/main/resources/PhenoTips/ExportPreferences.xml

```diff
@@ -119,34 +121,17 @@
         &lt;/li&gt;
       #end
       #foreach ($field in $xclass.properties)
-        #if (($activeFields.size() == 0 || $activeFields.contains(${field.name})) &amp;&amp; !$ignored.fields.contains(${field.name}) &amp;&amp; !$ignored.types.contains(${field.type}))
+        #if (($activeFields.size() == 0 || $activeFields.contains(${field.name})) &amp;&amp; (${field.name} != ${negPheno} &amp;&amp; ${field.name} != ${negPrenatal})&amp;&amp; !$ignored.fields.contains(${field.name}) &amp;&amp; !$ignored.types.contains(${field.type}))
         &lt;li&gt;
           &lt;label for="${targetFieldName}_${field.name}"&gt;
           &lt;input type="checkbox" name="${targetFieldName}" id="${targetFieldName}_${field.name}" value="${field.name}" #if($valueList.contains(${field.name}) || $valueListisEmpty) checked="checked" #end /&gt;
           ${field.translatedPrettyName}
         &lt;/li&gt;
-        #if ($codeFields.contains(${field.name}) &amp;&amp; !${simple})
-        &lt;li&gt;
-          &lt;label for="${targetFieldName}_${field.name}_code"&gt;
-          &lt;input type="checkbox" name="${targetFieldName}" id="${targetFieldName}_${field.name}_code" value="${field.name}_code" #if($valueList.contains("${field.name}_code")) checked="checked" #end /&gt;
-          ${field.translatedPrettyName} $services.localization.render('phenotips.exportPreferences.field.ID')
-        &lt;/li&gt;
-        &lt;li&gt;
-          &lt;label for="${targetFieldName}_${field.name}_combined"&gt;
-          &lt;input type="checkbox" name="${targetFieldName}" id="${targetFieldName}_${field.name}_combined" value="${field.name}_combined" #if($valueList.contains("${field.name}_combined")) checked="checked" #end /&gt;
-          ${field.translatedPrettyName} $services.localization.render('phenotips.exportPreferences.field.IDandName')
-        &lt;/li&gt;
-        #end
         #if ($metaFields.contains(${field.name}) &amp;&amp; !${simple})
         &lt;li&gt;
-          &lt;label for="${targetFieldName}_${field.name}_code_meta"&gt;
-          &lt;input type="checkbox" name="${targetFieldName}" id="${targetFieldName}_${field.name}_code_meta" value="${field.name}_code_meta" #if($valueList.contains("${field.name}_code_meta")) checked="checked" #end /&gt;
-          ${field.translatedPrettyName} $services.localization.render('phenotips.exportPreferences.field.termAndMetaID')
-        &lt;/li&gt;
-        &lt;li&gt;
           &lt;label for="${targetFieldName}_${field.name}_meta"&gt;
           &lt;input type="checkbox" name="${targetFieldName}" id="${targetFieldName}_${field.name}_meta" value="${field.name}_meta" #if($valueList.contains("${field.name}_meta") || $valueListisEmpty) checked="checked" #end /&gt;
-          ${field.translatedPrettyName} $services.localization.render('phenotips.exportPreferences.field.termAndMetaNames')
+          ${field.translatedPrettyName} $services.localization.render('phenotips.exportPreferences.field.qualifiers')
```

Comment:
This kind of translation chunking doesn't work, not all languages use the same topic. The full sentence should be translated.